src/postgraphile/postgraphile.ts

```diff
@@ -137,7 +152,7 @@ export function getPostgraphileSchemaBuilder<
             const diff = process.hrtime(start);
             const dur = diff[0] * 1e3 + diff[1] * 1e-6;
             if (!retry) {
-              releaseOnce();
+              await releaseOnce();
```

Comment:
If `releaseOnce` were to throw an error, this new error would be raised to the user rather than the original error, and the new error will contain less useful/addressable information. Instead we should swallow any new errors and continue to throw the old.

```suggestion
              // Swallow new error so old error is still thrown
              await releaseOnce().catch(e => {console.error(e)});
```